### DISCUSSION TITLE
How would you arrange your displays when you have 4 monitors?

### DISCUSSION THREAD

**Discussion by wheee** [nikitabobko/AeroSpace#901] [questions-and-answers]
**Describe the bug**
I have 4 monitors arranged like so:
![image](https://github.com/user-attachments/assets/f0083117-b554-42d7-856d-0fdd371017d8)

There's no way to configure these monitors to have an "open" bottom-right/left corner.

Is there a way to use top-right/left corners when appropriate?  

**Related issues you could find**
Couldn't find any.  I suspect that my setup is rather rare, but this is a deal breaker for me as I'm a heavy user of spaces (or workspaces).

Thanks!

**Comment by nikitabobko** [comment]
> Is there a way to use top-right/left corners when appropriate?

It's not possible - macOS restriction. `docs/guide.adoc` contribution to mention this fact is welcome
